docs/tutorials/ifci_px4_flight_controller.rst

```diff
@@ -0,0 +1,276 @@
+
+For each module ID that you have connected, set the following parameters to your desired values: ``CONTROL_MODE``, ``VERTIQ_FC_DIR``, ``VERTIQ_MOTOR_DIR``. Additionally for each module set the ``THROTTLE_CVI`` parameter to a different value. CVI stands for Control Value Index and more information can be found in :ref:`controlling_ifci`. Matching the module ID is a good way to do this, but if your module IDs do not start at 0, it is recommended that your CVIs do. If ``CONTROL_MODE`` is set to Voltage or Velocity, set the corresponding ``MAX_VOLTAGE`` or ``MAX_VELOCITY`` parameter as well.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/module_parameters.png
+    :align: center
+    :scale: 50
+    :alt: PX4 Module Parameters
+
+    PX4 Module Parameters
+
+Once these are all set, set the ``VERTIQ_NUM_CVS`` parameter to the highest CVI that you used plus 1. This refers to the number of Control Values the flight controller should send. If you only have 4 control values, but the highest one is 11, the flight controller needs to send 12 control values, even if some intermediate ones are unused. It better to use every control value you are sending to reduce overhead on the serial line. If you are doing a quadcopter and used ``THROTTLE_CVI`` 0, 1, 2, 3, you should set the ``VERTIQ_NUM_CVS`` to 4.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/num_cvs.png
+    :align: center
+    :scale: 50
+    :alt: Num CVs
+
+    Setting the Number of Control Values
+
+The final values to set are ``TELEM_IDS_1`` and ``TELEM_IDS_2``. These control which module IDs have telemetry requested. When you click on one of these parameters, a list of checkboxes should show up. Tick each module ID that is on your bus to request telemetry from that module.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/telem_ids.png
+    :align: center
+    :scale: 50
+    :alt: Telem Modules
+
+    Setting the Modules to Request Telemetry From
+
+Actuator Setup
+==============
+
+The next step is to set up the actuator associations. Go to the actuator tab in QGroundControl settings and select the 'Vertiq IO' tab on the top right. Initially every ESC function will be disabled. Each ESC number in this section corresponds to a CVI number plus 1. For example ESC 1 corresponds to CVI 0 and so on. Based on the geometry drawing, match the motor with the appropriate ESC # (and therefore CVI). As you do, also ensure that the direciton checkbox is ticked properly. Module ID is irrelevant here.
```

Comment:
on the top right...it's really just in the mididle

How about just "Under Actuator Outputs"